source/code/plugins/patch_management_lib.rb

```diff
@@ -230,8 +254,8 @@ def self.transform_and_wrap(inventoryXMLstr, host, time, force_send_run_interval
         availableUpdatesXML = instancesXML.select { |instanceXML| isAvailableUpdateInstanceXML(instanceXML) }
 
         # Convert to xml to hash/json representation
-        installedPackages = installedPackagesXML.map { |installedPackage|  installedPackageXMLtoHash(installedPackage, osShortName)}
-        availableUpdates = availableUpdatesXML.map { |availableUpdate|  availableUpdatesXMLtoHash(availableUpdate, osShortName)}
+        installedPackages = installedPackagesXML.map! { |installedPackage|  installedPackageXMLtoHash(installedPackage, osShortName)}
```

Comment:
Create an issue to to track the fix https://github.com/Microsoft/OMS-Agent-for-Linux/issues/268